dashboard/app/handler.go

```diff
@@ -68,7 +71,10 @@ func handleContext(fn contextHandler) http.Handler {
 }
 
 type (
-	ErrDontLog  struct{ error }
+	ErrDontLog struct {
+		error
+		HTTPStatus int // like http.StatusInternalServerError, etc
+	}
```

Comment:
Maybe it would be better to just rename `ErrDontLog` to `ErrNotFound`? It's anyway only used in this context, as I see. And it's quite reasonable that we don't log 404 errors.

In this case we also don't need the `HTTPStatus` field.

Reply:
RFC2616 call them "Client Error 4xx". Lets also use ClientError name.
We'll not log all the ClientErrors then.

We currently use 403 (Forbidden) and I saw places for 400 (Bad request). Want to use Status for other codes.